src/gluonts/distribution/gamma.py

```diff
@@ -122,8 +127,10 @@ def domain_map(cls, F, alpha, beta):
             Two squeezed tensors, of shape `(*batch_shape)`: both have entries mapped to the
             positive orthant.
         """
-        alpha = softplus(F, alpha) + 1e-8
-        beta = softplus(F, beta) + 1e-8
+        epsilon = np.finfo(cls._dtype).eps  # machine epsilon
```

Comment:
Similarly to my comments in the other PR: I don’t like much this static _dtype property. I think we could use the dtype of alpha (we can assume this is the same as beta’s), what do you think?

The same applies to the other similar cases

Reply:
Resolving after offline discussion